xgi/readwrite/hif.py

```diff
@@ -6,71 +6,79 @@
 
 import json
 from collections import defaultdict
+from os.path import dirname, join
 
-from ..convert import to_bipartite_edgelist
-from ..core import DiHypergraph, Hypergraph, SimplicialComplex
-from ..utils import IDDict
+from ..convert import from_hif_dict, to_hif_dict
+from ..exception import XGIError
 
-__all__ = ["write_hif", "read_hif"]
+__all__ = ["write_hif", "write_hif_collection", "read_hif", "read_hif_collection"]
```

Comment:
no read_hif?

Reply:
? Not sure what you mean?

Reply:
whoops, it's there, for some reason I didn't see!